git-rebase--interactive.sh

```diff
@@ -72,6 +72,7 @@ collapse_todo_ids() {
 
 # Switch to the branch in $into and notify it in the reflog
 checkout_onto () {
+	comment_for_reflog start
```

Comment:
From the commit message this is added because "$GIT_SEQUENCE_EDITOR is not longer set at the first call of
checkout_onto()". So I would say that this change is related to the previous commit that does not set $GIT_SEQUENCE_EDITOR anymore at the first call of checkout_onto() and should be moved in that previous commit.

Reply:
It’s already in the commit that removes setup_reflog_action()